css/style.css

```diff
@@ -4,8 +4,251 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    overflow-x: hidden;
 }
 
+header.logo{
+    width: 30px;
+    height: 30px;
+    position: relative;
+    top: 30px;left: 120px;
+}
+
+.navbar{
+  color:rgb(88, 69, 69) 
+  
+   
+}
+.hamburger{
+  display: none;
+}
+
+  ul li a {
+    list-style-type: none;
+    font-size: 15px;
+    position: relative;
+    top: 0px;right: 100px;
+    text-decoration: none;
+    
+}
+  ul li a:hover
+  {
+  color: rgb(235, 106, 31);
+  }
+
+  li {
+    display: inline;
+    padding: 7px;
+    margin: 10px;
+    margin-top: 0;
+    float: right;
```

Comment:
Please don't use `float` rules in your CSS. Float is the old way of making layouts in CSS and it has now been replaced by Flexbox (which was introduced a few years ago). 
Set `display: flex` on the parent of the element (`ul` in this case)